types/ts4.1/type-specs.tsx

```diff
@@ -77,6 +77,8 @@ const invalidParamsWithGeneric: Params<{ id: number }> = { id: 13 }; // $ExpectE
   {({ rest }) => {
     const fn = (a: string) => "noop";
     fn(rest); // $ExpectError
+
+    return <></>;
```

Comment:
The previous definition was incorrect, but thanks to the fix I was able to catch it.